index.html

```diff
@@ -1,62 +1,199 @@
-    <link
-      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Odibee+Sans&family=Poppins&family=Roboto:wght@700&display=swap"
-      rel="stylesheet"
-    />
-  </head>
-  <body>
-    <main>
-      <header class="container">
-        <a href="#" class="logo"><img src="./Logo_placeholder.png" alt="Logo"/></a>
-        <nav>
-          <a href="#" class="menu"><img src="./Enabled.png" alt="Menu button"/></a>
-        </nav>
-      </header>
-      <section class="headline">
-        <h1 class="primary-text">Hey There. I'm Festus</h1>
-        <h2 class="supporting-text">I am a Software Developer</h2>
-        <p class="description">
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don't hestiate to contact me.
-        </p>
-        <ul class="social-media">
-          <li>
-            <a href="#" class="git"
-              ><img src="./Github.png" alt="Github logo"
-            /></a>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <title>Personal Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Odibee+Sans&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <main>
+    <header class="container">
+      <a href="#" class="logo"><img src="./img/Logo_placeholder.png" alt="Logo" /></a>
+      <nav>
+        <a href="#" class="menu"><img src="./img/Enabled.png" alt="Menu button" /></a>
+      </nav>
+    </header>
+    <section class="headline">
+      <h1 class="primary-text">Hey There. I'm Festus</h1>
+      <h2 class="supporting-text">I am a Software Developer</h2>
+      <p class="description">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don't hestiate to contact me.
+      </p>
+      <ul class="social-media">
+        <li>
+          <a href="#" class="git"><img src="./img/Github.png" alt="Github logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="linkedin"><img src="./img/linkedIn.png" alt="Linkedin logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="angelist"><img src="./img/angelist.png" alt="Angelist logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="twitter"><img src="./img/twitter.png" alt="Twitter logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="vector"><img src="./img/vector.png" alt="Vector logo" /></a>
+        </li>
+      </ul>
```

Comment:
Great work so far @Enning94 🤝

- [ ] Please try to align this social media `<ul>` according to the Figma design provided.
- Your design
![your design](https://user-images.githubusercontent.com/63917725/176987883-fbe1d209-c93d-4bf6-a708-d7691e06b827.PNG)

- Figma design
![figma design](https://user-images.githubusercontent.com/63917725/176987890-0d40b08a-10db-47d4-8807-91ab3fc01d79.PNG)